server/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -569,6 +568,21 @@ private CompletionStage<Result> renderErrorOrRedirectToNextBlock(
                     applicantId, programId, nextBlockIdMaybe.get())));
   }
 
+  private boolean shouldRenderIneligibleBlockView(
```

Comment:
**suggestion:** Would you mind adding a comment here too similar to below explaining the cases where we should / shouldn't render this view?